conans/client/tools/system_pm.py

```diff
@@ -126,6 +126,49 @@ def install(self, packages, update=True, force=False, arch_names=None):
             self.update()
         self._install_any(packages)
 
+    def install_packages(self, packages, update=True, force=False, arch_names=None):
+        """ Get the system package tool install command and install all packages or variants.
+            Inputs:
+            "pkg-variant1"  # (1) String input: Install only one package
+            ["pkg-variant1", "otherpkg", "thirdpkg"] # (2) List of strings: all must be installed
+            [("pkg-variant1", "pkg-variant2"), "otherpkg", "thirdpkg"] # (3) List with alternatives
+
+        :param packages: String/List/Tuple with all packages to be installed e.g. "libusb-dev libfoobar-dev"
+        :param update: Run update command before to install
+        :param force: Force installing all packages
+        :param arch_names: Package suffix/prefix name used by installer tool e.g. {"x86_64": "amd64"}
+        :return: None
+        """
+        packages = [packages] if isinstance(packages, str) else list(packages)
+        variants = list(filter(lambda x: isinstance(x, tuple), packages))
```

Comment:
Do we want to consider here a `list` too?

```python
package_install([['pkg1-var1', 'pkg1-var2'], 'pkg2'])

Reply:
That's a good question. I think only tuple keep more explicit, but on the other hand, people need to read to documentation to understand that only tuple is allowed. I think accepting list should not be a problem, I'll update it.